javascript/javascript1/week3/Javascript1_Week3.js

```diff
@@ -0,0 +1,166 @@
+
+//Remove an item from an array: 
+  "use strict";
+  const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  
+const nameToRemove = "Ahmad"
+  function findIndex(nameToRemove) { 
```

Comment:
findIndex is a function of javascript that I think you will learn more about in js2. 
try to avoid the name of the javascript methods or keywords.